x-pack/platform/packages/shared/kbn-streams-schema/src/models/group/index.ts

```diff
@@ -11,14 +11,40 @@ import { validation } from '../validation/validation';
 import type { ModelValidation } from '../validation/model_validation';
 import { modelValidation } from '../validation/model_validation';
 
+interface StreamRelationship {
+  name: string;
+  type: 'related' | 'member';
+  filter?: string;
+}
+
+const streamRelationshipSchema = z.object({
+  name: z.string(),
+  type: z.union([z.literal('related'), z.literal('member')]),
+  filter: z.string().optional(),
+});
+
 export interface Group {
-  members: string[];
+  type: 'group';
```

Comment:
is this property redundant ? when we have access to it we should already know it's a group stream, but it could be useful at the root level

Reply:
> all groups created are just "group"

is the plan for group streams to eventually be a higher level concept similar to ingest ?